packages/block-editor/src/components/inserter/menu.js

```diff
@@ -249,19 +252,21 @@ function InserterMenu(
 								__experimentalInsertionIndex
 							}
 							showBlockDirectory
+							showBlocks={ ! isZoomOutMode }
 							shouldFocusBlock={ shouldFocusBlock }
 						/>
 					</div>
 				) }
 				{ showAsTabs && (
 					<InserterTabs
+						showBlocks={ ! isZoomOutMode }
```

Comment:
How about we set a new `const showBlocks = ! isZoomOutMode` and use that in these places that are now relying on `isZoomOutMode` where we're deciding if we should show blocks or not? It would better follow the existing pattern

Reply:
Thanks, done in 25121d0c31